test/aleph/http_test.clj

```diff
@@ -192,9 +195,12 @@
      (with-server (http/start-server ~handler {:port port :compression-level 3})
        ~@body)))
 
-(defmacro with-ssl-handler [handler & body]
-  `(with-server (http/start-server ~handler {:port port, :ssl-context (netty/self-signed-ssl-context)})
-     ~@body))
+(def default-ssl-options {:port port, :ssl-context (netty/self-signed-ssl-context)})
+
+(defmacro with-ssl-handler
+   [handler options & body]
+   `(with-server (http/start-server ~handler ~options)
```

Comment:
Shouldn't `with-ssl-handler` include the SSL options? Otherwise, it's the same as `with-server`